src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx

```diff
@@ -185,25 +179,16 @@ const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playl
         ))}
         {channelMediaItem ? <script type="application/ld+json">{generateMovieJSONLD(channelMediaItem)}</script> : null}
       </Helmet>
-      <VideoLayout
-        inlineLayout={false}
-        isLoading={false}
-        // live channels are public, so we don't need to check for entitlements
-        isLoggedIn={true}
-        hasSubscription={true}
-        accessModel={accessModel}
-        title={videoDetails.title}
-        description={videoDetails.description}
-        item={channelMediaItem}
-        primaryMetadata={primaryMetadata}
-        image={videoDetails.image}
-        startWatchingButton={startWatchingButton}
-        shareButton={shareButton}
-        trailerButton={null}
-        favoriteButton={null}
-        getURL={() => ''} // TODO: OWA-31 - Add link for EPG item
-        player={
-          channelMediaItem && (
+      <div className={styles.videoCinemaLayout} data-testid={testId('cinema-layout')}>
```

Comment:
Can you put this `div` inside of `VideoDetails` so it's not duplicated here and in `VideoLayout`? You can also then move the `videoCinemaLayout` style from `VideoLayout` to `VideoDetails`.

Reply:
It looks like there are no styles attached to the `videoCinemaLayout`, so I'll move it inside `VideoDetails` for the `data-testid `